src/models/session/dissolutionSession.model.ts

```diff
@@ -7,7 +8,10 @@ export default interface DissolutionSession {
   companyNumber?: string
   selectDirectorForm?: SelectDirectorFormModel
   selectSignatoriesForm?: SelectSignatoriesFormModel
-  directorsToSign?: DirectorToSign[],
-  applicationReferenceNumber?: string,
+  defineSignatoryInfoForm?: DefineSignatoryInfoFormModel
+  directorsToSign?: DirectorToSign[]
+  applicationReferenceNumber?: string
   approval?: DissolutionApprovalModel
+  isMultiDirector?: boolean
+  isApplicantADirector?: boolean
```

Comment:
added to support back link work so that they handle both single and multi director journeys